[PC-325] [WIP] Windows port (#55)
KanoComputing/mercury: README.rst

   pip3 install -r requirements-dev.txt
   apt-get install cmake doxygen lcov qt

Windows
-----

You will need a Windows 10 system, along with the additional software components:

- Cmake, version 3.15.4 or above: https://cmake.org/download
- Conan software: https://conan.io/downloads.html
- Git Client, command line version with shell: https://git-scm.com/download/win
- WinRAR for uncompressing the Swig software: https://www.win-rar.com
- Swig software for windows, zip file: http://www.swig.org/download.html

> - Developer Studio at least version 2019, with the following features:

Reviewer: ```suggestion
- Visual Studio at least version 2019, with the following features:
```

If your linux system is based on Debian Stretch, you can use the `backports <https://backports.debian.org/Instructions/>`_ repositories to pull the latest CMake version. Alternatively, you can point your APT sources to Debian Buster.

> On prior Buster Debian distros, you can run this code snippet as root to pull a comfortable CMake version:

Reviewer: ```suggestion
On prior Debian Stretch distros, you can run this code snippet as root to pull a comfortable CMake version:
```

.. code:: bash

   printf "deb http://deb.debian.org/debian stretch-backports main" > /etc/apt/sources.list.d/backports.list
   curl -L http://ftp.debian.org/debian/dists/stretch-backports/Release.gpg | apt-key add -
   apt-get update
   apt-get install --allow-unauthenticated -y -t stretch-backports cmake

> On Linux

Reviewer: ```suggestion
On Windows
```

--------
Open a Git Client Shell, clone this repository and then execute the batch file `windows-build.bat`.
Once the batch file terminates, you should see the file `ALL_BUILD.vcxproj` amongst other ones.

> You can either open this file inside Developer Studio, and follow the build targets interactively,

Reviewer: ```suggestion
You can either open this file inside Visual Studio, and follow the build targets interactively,
```

> or build it from the command line using `msdev`, by opening a PowerShell Developer Studio Command Prompt:

Reviewer: ```suggestion
or build it from the command line using `msdev`, by opening a Windows PowerShell:
```
